localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,390 @@
+ high_kingdom_empire:0 "High Kingdom"
+
+ # Republic
+ magistrate_male_duke:0 "Magistrate"
+ magistrate_female_duke:0 "Magistrate"
+ magistracy_duchy:0 "Magistracy"
+
+ ### Forsaken ###
+ # Republic
+ magistrate_male_forsaken_duke:0 "Magistrate"
+ magistrate_female_forsaken_duke:0 "Magistrate"
+ magistracy_forsaken_duchy:0 "Magistracy"
  
- warchief_male:0 "Warchief"
- warchief_female:0 "Warchief"
- warchief_orc_male:0 "Warchief"
- warchief_orc_female:0 "Warchief"
- warlord_male:0 "Warlord"
- warlord_female:0 "Warlord"
- high_overlord_male:0 "High Overlord"
- high_overlord_female:0 "High Overlord"
- overlord_male:0 "Overlord"
- overlord_female:0 "Overlord"
-
- king_nightborne_male:0 "King"
- queen_nightborne_female:0 "Queen"
-
- grand_prince_nightborne_male:0 "Grand Prince"
- grand_princess_nightborne_female:0 "Grand Princess"
- grand_principality_nightborne:0 "Grand Principality"
-
- prince_nightborne_male:0 "Prince"
- princess_nightborne_female:0 "Princess"
- principality_nightborne:0 "Principality"
-
- lord_nightborne_male:0 "Lord"
- lady_nightborne_female:0 "Lady"
+ ### Stromic ###
+ # Special
+ emperor_arathor_male:0 "Emperor"
+ emperor_arathor_female:0 "Empress"
+ empire_arathor:0 "Empire"
 
+ # Feudal
+ emperor_male_stromic:0 "Emperor"
+ emperor_female_stromic:0 "Empress"
+ empire_stromic:0 "Empire"
+
+ ### Kul'tiran ###
+ # Feudal
+ lord_male_tirassian_duke:0 "Lord"
+ lady_female_tirassian_duke:0 "Lady"
+ admiral_male_tirassian_king:0 "Admiral"
+ admiral_female_tirassian_king:0 "Admiral"
+ admiralty_tirassian_kingdom:0 "Admiralty"
+ grand_admiral_male_tirassian_emperor:0 "Grand Admiral"
+ grand_admiral_female_tirassian_emperor:0 "Grand Admiral"
+ grand_admiralty_tirassian_empire:0 "Grand Admiralty"
+
+ ### Zandalari ###
+ # Feudal
+ god_king_feudal_emperor:0 "God King"
+ god_queen_feudal_emperor:0 "God Queen"
+
+ prince_male_zandalari_kingdom:0 "Prince"
+ princess_female_zandalari_kingdom:0 "Princess"
+ prince_male_zandalari_empire:0 "Prince"
+ princess_female_zandalari_empire:0 "Princess"
+
+ ### Blood trolls ###
+ # Feudal
  blood_troll_male:0 "#"
- blood_troll_count_female:0 "Warmother"
- blood_troll_duke_female:0 "Grand Ma'da"
- blood_troll_queen_female:0 "Queen Ma'da"
- blood_troll_empress_female:0 "Divine Ma'da" lady_nightborne_female:0 "Lady"
+ warmother_female_blood_troll_count:0 "Warmother"
+ grand_mada_female_blood_troll_duke:0 "Grand Ma'da"
+ queen_mada_female_blood_troll_king:0 "Queen Ma'da"
+ divine_mada_female_blood_troll_emperor:0 "Divine Ma'da" 
  
- lich_king_male:0 "Lich King"
- lich_king_female:0 "Lich Queen"
+ ### Drakkari ###
+ # Feudal
+ frost_king_feudal_king:0 "Frost King"
+ frost_queen_feudal_king:0 "Frost Queen"
+
+ ### Dragons ###
+ # Feudal
+ overlord_male_dragon_duke:0 "Overlord"
+ overlady_female_dragon_duke:0 "Overlady"
+ lord_male_dragon_king:0 "Lord"
+ lady_female_dragon_king:0 "Lady"
+ dragonlord_male_dragon_emperor:0 "Dragon Lord"
+ dragonlady_female_dragon_emperor:0 "Dragon Lady"
```

Comment:
There are no high-ranking dragons named lords.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/135694386-6f21a4d4-9360-4dae-9f8d-670c4189190d.png)

</details>
https://wowpedia.fandom.com/wiki/List_of_aristocracy#Dragon_4

Reply:
Nefarian, Sintharia, and Onyxia.
Malygos is called Lord of Magic too https://wowpedia.fandom.com/wiki/Malygos
Ysera is called Lady of Dreams https://wowpedia.fandom.com/wiki/Ysera
Nozdormu is called Lord of Time https://wowpedia.fandom.com/wiki/Nozdormu
After as I said above, I planned that the aspects are called Aspects with their trait.
I can't say Emperor = Aspect because a dragon emperor may not be Aspect. -> Nefarian

Reply:
Copy our titles from CK2.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/135867522-306d172e-a867-4be8-b939-560106e063a0.png)

</details>

Reply:
Not convinced but done in ebda63f45